app/models/user.rb

```diff
@@ -20,7 +20,10 @@ def self.find_or_create_by_oauth(oauth)
     user.image_url = oauth.info.image
     user.token = oauth.credentials.token
     user.github = oauth.info.urls.GitHub
-    user.location = Octokit.user( user.nickname).location
+    if ENV['RAILS_ENV'] != 'test'
+      user.location = Octokit.user( user.nickname).location
+    end
```

Comment:
Are we sure that we aren't going to need nickname sometime in the future for a test?